src/main/java/redis/clients/jedis/params/ZAddParams.java

```diff
@@ -66,6 +67,15 @@ public ZAddParams lt() {
     return this;
   }
 
+  /**
+   * Acts like ZINCRBY and only one score-element pair can be specified in this mode.
+   * @return ZAddParams
+   */
+  public ZAddParams incr() {
```

Comment:
`incr()` is not required here. Because in `zaddIncr(...)`, we already know that `INCR` argument is required.
Also, the added benefit of not adding `incr()` here is that user won't be able to use it in normal `zadd(...)`.

Reply:
👍 you were right. thanks for so quick reply. : )